### PR TITLE
Adds visual space for empty item slot #702

### LINKED ISSUE #702: Items in inventory are visually collapsed
For instance, my friend when he builds force staff, will always keep it in his 4th item slot.

In example: [Match 1848421474](http://yasp.co/matches/1848421474), he plays the Shadow Shaman. The TP boots are in item slot 1, so that looks correct. However the force staff is in the 4th item slot and the euls is in the 5th item slot.

So, we'd both like some spacers filling in the empty inventory slots. However, I don't know if the inventory slot index is currently tracked?

### PR DESCRIPTION
**Pull request by ahockersten**
fixes #702

### CODE REVIEW

**Comment by howardchung** [comment]
Looks okay, but number may need to be tweaked later (I changed some sizes in sql branch)

**Comment by howardchung** [comment]
@albertcui please review

File: views/match/match_index.jade [hunk 1]
```diff
@@ -62,6 +62,8 @@ block match_content
                   - var item = constants.items[constants.item_ids[val]]
                   if item
                     img.img-small.item(src=item.img, title=constants.item_ids[val])
+                  else
+                    span.spacer-small.item
```
Comment (howardchung) [inline views/match/match_index.jade]: This should also be done for lone druids bear

Comment (ahockersten) [inline views/match/match_index.jade]: Nice catch. I'm fixing this. However, I found another bug while doing so (reported as #709). I'll look into fixing that one as well as a part of this patch.

File: views/match/match_index.jade [hunk 2]
```diff
@@ -69,7 +71,9 @@ block match_content
                     - var item = constants.items[constants.item_ids[val]]                             
                     if item
                       img.img-small.ability(src=item.img, title=constants.item_ids[val])
```
Comment (howardchung) [inline views/match/match_index.jade]: Found the issue.  .ability should be .item

**Comment by albertcui** [comment]
I don't really like the spacer. I don't think which item slots are empty is useful information, and since Dotabuff also has what we're doing, I think it would be confusing for users to add the spacer.

I've added the commit for fixing #709 though. Thanks!

**Pull request closed without merging**